packages/backend/src/peripherals/database/migrations/080_timestamp_column_tz.ts

```diff
@@ -0,0 +1,59 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server.
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+interface ColumnInfo {
+  table_catalog: string
+  table_schema: string
+  table_name: string
+  column_name: string
+  data_type: string
+}
+
+const table_schema = 'public'
+const tableWhitelist = [
+  'circulating_supplies',
+  'total_supplies',
+  'total_supplies_status',
+  'update_notifier',
+]
+
+export async function up(knex: Knex) {
+  for (const table_name of tableWhitelist) {
+    // get all columns that are of type timestamp with time zone
+    const columns = (await knex('information_schema.columns')
```

Comment:
This line introduced bug previously, although now we have whitelist I would like the code to be even more explicit. Please change the `tableWhitelist` to `tablesToUpdate` and just iterate over them and on each step run the raw query to change the column type